docs/release-notes/2.40.1.md

```diff
@@ -0,0 +1,14 @@
+# Percona Monitoring and Management 2.40.1
+
+| **Release date:** | Oct 18, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issues
+
+- [PMM-12592](https://jira.percona.com/browse/PMM-12592) - Due to a known issue, the **Library panels** on the PMM dashboard were not working. The issue has been resolved now. 
+- [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading PMM from version 2.39.0 to 2.40.0 (not el7) using Docker, the admin user could not access the PMM UI. The issue has been resolved now.
+- [PMM-12587](https://jira.percona.com/browse/PMM-12587) - During the upgrade from version 2.39 to 2.40, the process was failing because of incorrect mappings between dashboards, folders, users, and groups. The issue has been resolved.
```

Comment:
```suggestion
- [PMM-12587](https://jira.percona.com/browse/PMM-12587) - During the upgrade from version 2.39.0 to 2.40.0, the process was failing because of incorrect mappings between dashboards, folders, users, and groups. The issue has been resolved.
```